cmake/HPX_AddConfigTest.cmake

```diff
@@ -208,7 +209,7 @@ macro(hpx_check_for_cxx11_auto)
   add_hpx_config_test(HPX_WITH_CXX11_AUTO
     SOURCE cmake/tests/cxx11_auto.cpp
     FILE ${ARGN}
-    CMAKECXXFEATURE cxx_auto_type)
+    CMAKECXXFEATURE cxx11_auto_type)
```

Comment:
There is no C++ feature detection property in cmake called `cxx11_auto_type` (please see https://cmake.org/cmake/help/v3.9/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html)